frontend/src/scenes/session-recordings/playlist/Playlist.tsx

```diff
@@ -132,90 +139,139 @@ export function Playlist({
-                                    defaultActiveKeys={openSections}
-                                    panels={sections.map((s) => {
-                                        return {
-                                            key: s.key,
-                                            header: s.title ?? '',
-                                            content: (
-                                                <SectionContent
-                                                    section={s}
-                                                    loading={!!loading}
-                                                    setActiveItemId={onChangeActiveItem}
-                                                    activeItemId={activeItemId}
-                                                    emptyState={listEmptyState}
-                                                />
-                                            ),
-                                            className: 'p-0',
-                                        }
-                                    })}
-                                    onChange={onChangeOpenSections}
-                                    multiple
-                                    embedded
-                                    size="small"
-                                />
-                            ) : sectionCount === 1 ? (
-                                <SectionContent
-                                    section={sections[0]}
-                                    loading={!!loading}
-                                    setActiveItemId={onChangeActiveItem}
-                                    activeItemId={activeItemId}
-                                    emptyState={listEmptyState}
-                                />
-                            ) : loading ? (
-                                <LoadingState />
-                            ) : (
-                                listEmptyState
-                            )}
-                        </div>
-                        <div className="shrink-0 relative flex justify-between items-center gap-0.5 whitespace-nowrap">
-                            {footerActions}
-                        </div>
-                    </div>
+                <div className="flex justify-end">
+                    <LemonButton
+                        icon={<IconX />}
+                        onClick={() => {
+                            setIsExpanded(false)
+                            posthog.capture('ai_filter_close')
+                        }}
+                    />
                 </div>
+                <AiFilter />
             </div>
+
             <div
-                className={clsx('Playlist h-full min-h-96 w-full min-w-96 lg:min-w-[560px] order-first xl:order-none', {
-                    'Playlist--wide': size !== 'small',
-                    'Playlist--embedded': embedded,
+                className={clsx('flex flex-col w-full gap-2 h-full', {
```

Comment:
are we sure we're not using the wide and embedded classes? 
they might be necessary when shared or full screen?